memoize/__init__.py

```diff
@@ -1 +1 @@
-from .decorator import memoize
+from .decorator import memoize  # noqa: F401
```

Comment:
You could probably also fix this with an assigment to `__all__`. Haven't tested. Gotta go...